proxy/proxy/client.go

```diff
@@ -329,6 +329,12 @@ func needsRefresh(e cacheEntry, refreshCfgBuffer time.Duration) bool {
 }
 
 func (c *Client) cachedCfg(ctx context.Context, instance string) (string, *tls.Config, string, error) {
+	// Validate instance string in order to exclude potentially unwanted directories picked up by fuse.
```

Comment:
Here's another section that can be deleted (accidentally included during the rebase).